### PR TITLE
Update various quickstarts to mutiny

### PR DESCRIPTION
**Pull request by cescoffier**
**Checklist**:

Your pull request:

- [X] targets the `development` branch
- [X] uses the `999-SNAPSHOT` version of Quarkus
- [X] has tests (`mvn clean test`)
- [X] works in native (`mvn clean package -Pnative`)
- [X] has native tests (`mvn clean verify -Pnative`)
- [X] makes sure the documentation must not be updated
- [X] links the documentation update pull request (if needed)
- [X] updates or creates the `README.md` file (with build and run instructions)
- [X] For new quickstart, is located in the directory _component-quickstart_
- [X] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by gsmet** [comment]
Rebased.

File: context-propagation/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+quarkus.datasource.url=jdbc:postgresql://localhost/quarkus_test
```
Comment (kenfinnigan) [inline context-propagation/src/main/resources/application.properties]: I could be wrong, but doesn't this need mp config for the reactive connector type?

File: neo4j-quickstart/src/main/java/org/acme/neo4j/ReactiveFruitResource.java [hunk 2]
```diff
@@ -25,9 +25,15 @@
     @GET
     @Produces(MediaType.SERVER_SENT_EVENTS)
     public Publisher<String> get() {
-        return Flux.using(driver::rxSession, session -> session.readTransaction(tx -> {
-            RxResult result = tx.run("MATCH (f:Fruit) RETURN f.name as name ORDER BY f.name");
-            return Flux.from(result.records()).map(record -> record.get("name").asString());
-        }), RxSession::close);
+        return Multi.createFrom().resource(
```
Comment (kenfinnigan) [inline neo4j-quickstart/src/main/java/org/acme/neo4j/ReactiveFruitResource.java]: Wow, so much easier to read than the Flux version

File: context-propagation/src/main/resources/application.properties (hunk 1 above)
Comment (cescoffier) [reply context-propagation/src/main/resources/application.properties]: no, here we want to use the blocking one.

**Pull request merged**